installscripts/cookbooks/jenkins/recipes/configurejenkins.rb

```diff
@@ -92,84 +152,161 @@
+end
+
+#Set up Cognito creds
+cookbook_file "#{node['script_root']}/cognitouser.sh" do
+  source 'credentials/cognitouser.sh'
+  mode 0755
 end
 
 execute 'createcredentials-cognitouser' do
-  command "#{node['cookbook_root']}/jenkins/files/credentials/cognitouser.sh #{node['jenkinselb']} #{node['jenkins']['clientjar']} #{node['authfile']}"
+  command "#{node['script_root']}/cognitouser.sh #{node['jenkins']['clicommand']} #{node['cognitouser']} #{node['cognitopassword']}"
 end
 
-execute 'configJenkinsLocConfigXml' do
-  command "#{node['cookbook_root']}/jenkins/files/node/configJenkinsLocConfigXml.sh  #{node['jenkinselb']} #{node['jenkins']['SES-defaultSuffix']}"
+#Set up create-service job
+cookbook_file "#{node['script_root']}/job_create-service.sh" do
+  source 'jobs/job_create-service.sh'
+  mode 0755
 end
 
 execute 'createJob-create-service' do
-  command "#{node['cookbook_root']}/jenkins/files/jobs/job_create-service.sh #{node['jenkinselb']} #{node['jenkins']['clientjar']} #{node['authfile']} #{node['scmpath']}"
+  command "#{node['script_root']}/job_create-service.sh #{node['jenkins']['clicommand']} #{node['scmpath']}"
+end
+
+#Set up delete-service job
+cookbook_file "#{node['script_root']}/job_delete-service.sh" do
+  source 'jobs/job_delete-service.sh'
+  mode 0755
 end
 
 execute 'createJob-delete-service' do
-  command "#{node['cookbook_root']}/jenkins/files/jobs/job_delete-service.sh #{node['jenkinselb']} #{node['jenkins']['clientjar']} #{node['authfile']} #{node['scmpath']}"
+  command "#{node['script_root']}/job_delete-service.sh #{node['jenkins']['clicommand']} #{node['scmpath']}"
+end
+
+#Set up build_pack_api job
+cookbook_file "#{node['script_root']}/job_build_pack_api.sh" do
+  source 'jobs/job_build_pack_api.sh'
+  mode 0755
 end
 
 execute 'createJob-job_build_pack_api' do
-  command "#{node['cookbook_root']}/jenkins/files/jobs/job_build_pack_api.sh #{node['jenkinselb']} #{node['jenkins']['clientjar']} #{node['authfile']} #{node['scmpath']}"
+  command "#{node['script_root']}/job_build_pack_api.sh #{node['jenkins']['clicommand']} #{node['scmpath']}"
 end
 
-execute 'createJob-bitbucketteam_newService' do
-  only_if { node['scm'] == 'bitbucket' }
-  command "#{node['cookbook_root']}/jenkins/files/jobs/job_bitbucketteam_newService.sh #{node['jenkinselb']} #{node['jenkins']['clientjar']} #{node['authfile']} #{node['scmelb']}"
+if node['scm'] == 'bitbucket'
+  #Set up bitbucketteam_newService job
+  cookbook_file "#{node['script_root']}/job_bitbucketteam_newService.sh" do
+    source 'jobs/job_bitbucketteam_newService.sh'
+    mode 0755
+  end
+
+  execute 'createJob-bitbucketteam_newService' do
+    command "#{node['script_root']}/job_bitbucketteam_newService.sh #{node['jenkins']['clicommand']} #{node['scmelb']}"
+  end
 end
 
-execute 'createJob-platform_api_services' do
-  only_if { node['scm'] == 'bitbucket' }
```

Comment:
@suryajak We dont need "Jazz_Core_Services" for scenario3 right? if so we need this checking otherwise an empty Jazz_Core_Services will be created for stack created for scenario3
@bleggett

Reply:
@rajeevr2715 So you're saying we *only* need to run `job_platform_api_services.sh` if the SCM is bitbucket?

Reply:
Unfortunately yes. We don't create certain folders like "Jazz_Core_Services" and "Jazz_User_Services" in scenario 3 (it has to do with how gitlab webhooks work).

Reply:
Alright, changed it so that step is only run if scm==bitbucket, think it's correct now.